modules/RTC/RTC.js

```diff
@@ -260,11 +260,17 @@ export default class RTC extends Listenable {
                     this._pinnedEndpoint);
                 this._channel.sendSelectedEndpointMessage(
                     this._selectedEndpoint);
+
+                if (typeof this._maxFrameHeight !== 'undefined') {
+                    this._channel.sendReceiverVideoConstraintMessage(
+                        this._maxFrameHeight);
```

Comment:
should `this._maxFrameHeight` be cleared after it's been successfully sent? (either via the cache or normally?)

Reply:
I don't think it needs to be cleared, so in this case I don't think it should. The way the caching works is the last selected maxFrameHeight is always saved when the maxFrameHeight is set, so it's essentially a reference to the last selected maxFrameHeight but its usage happens to be for sending the selected frame height on channel open. If I cleared it here, it would get set again when the client sets a new maxFrameHeight, unless I want to conditionally cache.